### ISSUE TITLE
mod compatibility issue

### ISSUE THREAD

**Issue by IchigoGames** [pWn3d1337/Techguns#152]
So i'm using this mod in a modpack i made but i also use galacticraft and as you can see in the screenshot this mod is trying to use the same spot as galacticraft in the inv is there anyway you could make it auto auto detect that there is already something in the spot and move over one.

https://imgur.com/a/PGi0h

techguns-1.12.2-2.0.1.1
mc 1.12.2
Forge 2624

**Comment by randomei** [comment]
I also have this issue.

techguns-1.12.2-2.0.1.1
mc 1.12.2
Forge 2611

**Comment by mgrabov** [comment]
Same thing ! using:
GalacticraftCore-1.12.2-4.0.1.172
TechGuns-1.12.2-2.0.1.1

Heres a thread on the galacticraft git: 
https://github.com/micdoodle8/Galacticraft/issues/3152

This is critical and breaks compatibility! This mod is great and looks very promising and I would love to have it work properly with other mods!

**Comment by caligator** [comment]
yea- late october here, have same issue with the techguns tab's compatibility with galacticraft. searching around I found this on the GC git https://github.com/micdoodle8/Galacticraft/issues/3152

so this is a pretty old and substantiated error in you guys mod. heres the screencap from my game showcasing the error; https://puu.sh/BQ62i/ec2ac10731.png

I am hoping to use the mod for my upcoming Doom-based modpack, taking advantage of GC's air systems and the fact that they do have mars as a dimension for map building... but if the error persists I'll have to turn to another mod instead for weapons.

**Comment by pWn3d1337** [comment]
The pre-release versions for 2.0.1.2 have this fixed.
https://github.com/pWn3d1337/Techguns2/releases/tag/2.0.1.2_pre2

**Comment by IchigoGames** [comment]
ok thats good in all but it does not help if its not on curse we still cant use it for modpacks.

**Comment by IchigoGames** [comment]
if you go by the last update on curse Jan 9, 2018 this mod is basically dead witch i hate cause is a really cool mod.

**Comment by pWn3d1337** [comment]
Yes the mod is currently close to dead. As I don't have much time for development.
